pkg/course/course.go

```diff
@@ -397,6 +398,23 @@ func (f FileV2) validateJsonSchema(schemaData []byte) error {
 	return nil
 }
 
+// SetGitClonePath allows the caller to set the path of a locally stored chart from a git repository already cloned
+func (r *Release) SetGitClonePath(path string) error {
```

Comment:
Let's call this when we read in the course file, in the NewClient() function from the reckoner package.